### ISSUE TITLE
Use google icons not fa-awesome!

### ISSUE THREAD

**Issue by rubenstolk** [pkaske/pk-datepicker#3]
Why use font-awesome if MD has nice icons...

**Comment by trading-peter** [comment]
I originally build the component for a existing project that's using font awesome in and out.
To match the look and feel I choose font awesome as well.
As for solutions to this "taste issue": Injecting the calendar controls or forking the project to swap the icons would help.
I don't have time to do this right now. Feel free to jump in if you want to help.

**Comment by rubenstolk** [comment]
https://github.com/changer/pk-datepicker

I think it's not really a taste issue for those 4 icons, I just think that iron-icon is better as it comes by default with Polymer.

**Comment by trading-peter** [comment]
Awesome! Thanks for the effort.
Well, good luck to win that discussion with a designer.
As said, I have to meet the design guidelines of the project I build the component for, so I need font awesome. Maybe I implement `<content>` injections from light DOM. This way everybody can use want he wants for calendar controls.

**Comment by rubenstolk** [comment]
Right :) So how are getting along with Polymer otherwise? I still find it quite a task compared to 0.5... Are you making a mobile app out of it?

## 

BR, Ruben Stolk
+919561050034  |  +31646118037

On Wed, Jun 24, 2015 at 9:20 AM -0700, "Peter Kaske" notifications@github.com wrote:

Awesome! Thanks for the effort.

Well, good luck to win that discussion with a designer.

As said, I have to meet the design guidelines of the project I build the component for, so I need font awesome. Maybe I implement <content> injections from light DOM. This way everybody can use want he wants for calendar controls.

**Comment by trading-peter** [comment]
I have to say, I'm still in the early stages of developing with polymer. I experimented with 0.5 but was too busy with another project to really dig deep. Thanks to that I haven't really had to unlearn the 0.5 way of polymer. That makes it easier I guess...
Right now I'm implementing smaller components into an existing project. But in parallel I started a bigger project where I want to go heavy on Polymer for front-end stuff. And I will try to make this app mobile friendly with the help of Polymer.
In any case I'm convinced that the concept of web-components is the future.
What part of Polymer bothers you the most?
